content/en/software/using-medley/keystrokes.md

```diff
@@ -25,15 +26,11 @@ We'd like to improve keyboard handling but haven't yet found a path.
 
 ## Interrupt characters
 
-These are enabled per-process. Medley maintains, for each process, a "termtable" which enables different kinds of interrupts. In general, the process (sometimes known as the "TTY" process or the process that "has the keyboard". 
-
-|character|action                                                |
-|---------|-----------------------------------------             |
-|ctrl-B   |stop the process and enter a break window             |
-|ctrl-D   | reset the process, unwind the stack to the top level |
-|ctrl-E   | unwind the stack as if an error occured without break|
-|ctrl-P   | interrupt printing and reset the PRINTLEVEL          |
-
-
-
+These are enabled per-process. Medley maintains, for each process, a "termtable" which enables different kinds of interrupts. In general, the process (sometimes known as the "TTY" process or the process that "has the keyboard".
```

Comment:
@masinter Can you take a look at this line and fix the last sentence?  It is incomplete.  Or, is it an artifact that should be deleted?

Reply:
The sentence is incomplete but I'm not sure it was ever right.  I wonder if there's a better explanation in the IRM?  The discussion about the relationship of interrupt characters, the termtable, and the proccess-with-tty seems at best incomplete to me now.

Reply:
I replaced the text with a block from the Medley IRM -- from page 29-1.  I think that gets at the desired point.

It also mentions a different control character, `ctrl-G`, 'Pops up a menu listing all of the currently-running processes. Selecting one of the processes will cause a break to take place in that process.' and states the `crtl-P` is no longer supported.

If this is correct, I'll go ahead and make those changes also.